src/main/java/org/broadinstitute/consent/http/enumeration/UserRoles.java

```diff
@@ -22,6 +21,18 @@ public enum UserRoles {
 
     private final String roleName;
     private final Integer roleId;
+    private static final HashSet<Integer> LIST_OF_NON_DAC_ROLE_IDS = new HashSet<>(Arrays.asList(
```

Comment:
Interestingly enough, I've looked at the Java SE Documentation for Java 17 and TIL:
Both List.of and Set.of allow the underlying elements to be modified, even though elements in the collection cannot be added, removed, or replaced.
See: https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/List.html and https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/Set.html
They're very careful to say the underlying elements may be mutable. `However, if the contained elements are themselves mutable, this may cause the List's contents to appear to change.`

For now, I think I'm going to use Set.of.  Nice suggestion Connor.